From 06c940f36e79be42e3f7e18166baf9232aed9023 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Thomas=20Wei=C3=9Fschuh?= <thomas@t-8ch.de>
Date: Fri, 10 Dec 2021 19:46:18 +0100
Subject: [PATCH 05/11] ao_pipewire: make opt-in

The original author of ao_pipewire has reported to experience vsync
jitter with this ao, so have the user opt-in.
---
 audio/out/ao.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/audio/out/ao.c b/audio/out/ao.c
index 8fc56ef860..b9c1176fd9 100644
--- a/audio/out/ao.c
+++ b/audio/out/ao.c
@@ -64,9 +64,6 @@ static const struct ao_driver * const audio_out_drivers[] = {
 #if HAVE_COREAUDIO
     &audio_out_coreaudio,
 #endif
-#if HAVE_PIPEWIRE
-    &audio_out_pipewire,
-#endif
 #if HAVE_PULSE
     &audio_out_pulse,
 #endif
@@ -91,6 +88,9 @@ static const struct ao_driver * const audio_out_drivers[] = {
 #endif
 #if HAVE_SDL2_AUDIO
     &audio_out_sdl,
+#endif
+#if HAVE_PIPEWIRE
+    &audio_out_pipewire,
 #endif
     &audio_out_null,
 #if HAVE_COREAUDIO
-- 
2.34.1

